.github/workflows/_lint.yml

```diff
@@ -14,6 +14,9 @@ env:
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      FETCH_DEPTH: 50
```

Comment:
This number is sort of arbitrarily set: we want it to be big enough that it's bigger than the number of commits in any reasonable PR, and also as small as possible since increasing the number makes the initial `git fetch` slower.

50 seemed like a reasonable middle ground.

Reply:
i often reach numbers in the 20-30, especially due to fighting with linters, but i think this is fine; would you mind adding this comment about how 50 right above the var value?

Reply:
Happy to. Also happy to increase it if you think it's worth it. Bumping it to ~100 only add another ~second or so.